docs/apache-airflow/issue-triage-process.rst

```diff
@@ -0,0 +1,195 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+
+Issue reporting and resolution process
+======================================
+
+This document explains the issue tracking and triage process within Apache
+Airflow including labels, milestones, and priorities as well as the process
+of resolving issues.
```

Comment:
Should we mention here that there is no need to open an issue prior to opening a PR?

Reply:
Agree. Good point. People do not realise that -it's a rare thing not having to open issue. I think we are pioneers with that. i actually find it pretty cool that we base all that we do on PR numbers rather than issue numbers. This is sooo much less overhead, but people do not realise that this is the case.

Actually we could even modify our template for new issues to include that information alongside "Please save some trees and do not create issue if you can just open PR fixing the problem" ;).

Reply:
Good point, done.